script/DeployBase.s.sol

```diff
@@ -401,10 +401,6 @@ abstract contract DeployBase is Script {
                 accounting.SET_BOND_CURVE_ROLE(),
                 address(vettedGate)
             );
-            accounting.grantRole(
```

Comment:
add role revoke to the upgrade vote script

Reply:
and keep the corresponding tests for it

Reply:
we 100% need to revoke this role from all members and check it in the vote tests to not keep the storage dirty

Reply:
Done